lib/cc/service/http.rb

```diff
@@ -21,10 +21,10 @@ def service_get(url = nil, body = nil, headers = nil, &block)
     raw_get(url, body, headers, &block)
   end
 
-  def service_post(url, body = nil, headers = nil, &block)
+  def service_post(url, body = nil, headers = nil, response_prefix = nil, &block)
```

Comment:
I went forward with extracting some response formatter classes... the nice thing is that we can just inject the formatter class, and so the http layer isn't concerned with formatting the response, it just uses the injected formatter to do that, which gives us flexiblity to have a few different kinds of formatters, which enables prefixing sometimes... WDYT?